src/index.html

```diff
@@ -0,0 +1,301 @@
+            </div>
+            <div>
+                <img class="icon bookmarks-icon" src="../assets/svg/bookmark-svgrepo-com.svg">
+            </div>
+            <div>
+                <img class="icon lists-icon" src="../assets/svg/square-list-svgrepo-com.svg">
+            </div>
+            <div>
+                <img class="icon profile-icon" src="../assets/svg/profile-about-mobile-ui-svgrepo-com.svg">
+            </div>
+            <div>
+                <img class="icon more-icon" src="../assets/svg/more-horizontal-circle-svgrepo-com.svg">
+            </div>
+        </section>
+    </section>
+
+    <section id="right-menu">
+        <header><input class="rounded-container" type="text" placeholder="Search Clone"></header>
+        <main>
+            <div class="categories">       
+                <h2>What's happening</h2>
+                <div></div>
+                <div></div>
+                <div></div>
+                <a>Show more</a>
+            </div>
+            <div class="categories">       
+                <h2>You might like</h2>
+                <div></div>
+                <div></div>
+                <div></div>
+                <div></div>
+                <a>Show more</a>
+            </div>
+            <div class="categories">       
+                <h2>What's happening</h2>
+                <div></div>
+                <div></div>
+                <div></div>
+                <a>Show more</a>
+            </div>
+        </main>
+    </section>
+    <!--<section id="message-box">Messages</section>-->
+    </div>
+    
+    <!--<footer>Koala svg by <a href="https://www.svgrepo.com/author/vectordoodle/">vectordoodle / </a> stormtrooper svg and deathstar svg by <a href="https://www.svgrepo.com/author/Good%20Stuff%20No%20Nonsense/">good stuff no nonsense / </a> hash svg by <a href="https://www.svgrepo.com/author/Denali%20Design/">Denali design / </a> bell svg by <a href="https://www.svgrepo.com/author/krystonschwarze/">krystonschwarze / </a>envelope svg by<a href="https://www.svgrepo.com/author/thewolfkit/">thewolfkit / </a> bookmarks svg and list svg by <a href="https://www.svgrepo.com/author/Dazzle%20UI/">Dazzle UI / </a>profile svg by <a href="https://www.svgrepo.com/author/kreevstudio/">kreevstudio / </a> more svg by <a href="https://www.svgrepo.com/author/scarlab/">scarlab</a> / more svg by <a href="https://www.svgrepo.com/author/Giovana/">Giovana</a>. Koala photo by <a href="https://www.pexels.com/photo/koala-bear-sleeping-on-tree-16904943/">Aisling Kerr</a> </footer>-->
+</body>
+</html>
+
+
+
+<!--☑ Layout the main page elements (left-hand menu, tweet feed and right side-bar) using CSS Grid. CSS Grid should also be used to layout individual tweets.
+☑ Define the page contents using HTML with appropriate elements. HTML should be well formatted.
+☑ Style the page using a mixture of Type, ID and Class CSS selector types
+☑ Implement hover and selected state for the left-hand menu links.-->
```

Comment:
Oooh very organised, my todos were just hidden around my code 😆

Reply:
honestly, that's just me copy-pasting the todos on the README files so I can glance at them without have to open said file - I'm trying to limit the number of tabs and windows I've got open at any given time, otherwise I lose track of some, then waste time trying to find info. Could blame that on my having only one monitor, but it's probably also got a lot to do with learning to organise stuff lol